common.mk

```diff
@@ -81,6 +86,11 @@ SOONG_CONFIG_NAMESPACES += qti_kernel_headers
 SOONG_CONFIG_qti_kernel_headers := version
 SOONG_CONFIG_qti_kernel_headers_version := $(SOMC_KERNEL_VERSION)
 
+# Build 64bit audio service
+SOONG_CONFIG_NAMESPACES += android_hardware_audio
+SOONG_CONFIG_android_hardware_audio := run_64bit
+SOONG_CONFIG_android_hardware_audio_run_64bit := true
```

Comment:
Do we also need `AUDIOSERVER_MULTILIB := first`? I used to have that when enabling 64-bit audio on my local device (but it was only used by soundtrigger iirc).

Reply:
It's finally not a thing anymore